tests/omprog-close-unresponsive-vg.sh

```diff
@@ -31,9 +31,7 @@ main_queue(
 }
 '
 startup_vg
-. $srcdir/diag.sh wait-startup
```

Comment:
do you really want it to run without knowing if it is read to run?

Reply:
I saw in diag.sh that 'startup' (and 'startup_vg') already does a 'wait-startup', so I removed this useless 'wait-startup' (when immediately following 'startup') in all tests. Similarly, I removed all 'wait-queueempty' just before 'shutdown_when_empty', for the same reason. Is that correct?

Reply:
ummm... sometimes I should re-read code that has evolved over the years. Quick check reveals you are right - I admit this wasn't on my mind. I did not question the existing tests and took for granted they are right in doing this sequence... Thanks for the wakeup-call!